docs/walkthrough.md

```diff
@@ -34,15 +34,14 @@ significantly different.
 In order to follow this walkthrough, you'll need container images for
 Prometheus and the custom metrics adapter.
 
-It's easiest to deploy Prometheus with the [Prometheus
-Operator](https://coreos.com/operators/prometheus/docs/latest/), which
+The [Prometheus Operator](https://coreos.com/operators/prometheus/docs/latest/),
 makes it easy to get up and running with Prometheus.  This walkthrough
 will assume you're planning on doing that -- if you've deployed it by hand
 instead, you'll need to make a few adjustments to the way you expose
 metrics to Prometheus.
 
-The adapter has different images for each arch, and can be found at
-`directxman12/k8s-prometheus-adapter-${ARCH}`.  For instance, if you're on
+The adapter has different images for each arch, which can be found at
+`directxman12/k8s-prometheus-adapter-${ARCH}`.  As an example, if you're on
```

Comment:
super nitpicky nit: I prefer "For instance" or "For example" here

Reply:
no worries :+1: